tests/python/rest_api/test_tasks.py

```diff
@@ -221,6 +221,23 @@ def test_can_remove_owner_and_fetch_with_sdk(self, admin_user, tasks):
         source_task["owner"] = None
         assert DeepDiff(source_task, fetched_task, ignore_order=True) == {}
 
+    @pytest.mark.usefixtures("restore_db_per_function")
+    def test_check_task_status_after_changing_job_state(self, admin_user, tasks, jobs):
+        task = next(t for t in tasks if t["jobs"]["count"] == 1 if t["jobs"]["completed"] == 0)
+        job = next(j for j in jobs if j["task_id"] == task["id"])
+
+        with make_api_client(admin_user) as api_client:
+            api_client.jobs_api.partial_update(
+                job["id"],
+                patched_job_write_request=models.PatchedJobWriteRequest(
+                    state="completed", stage="acceptance"
+                ),
```

Comment:
It does not really test original bug.
Please, refer to the issue.

The problem occured when we patch `stage` to `acceptance` and in another request we patch `state` to `completed`.

Reply:
If to patch both in signle request, it worked well.